### PR TITLE
Kf 6 add sp link

### PR DESCRIPTION
**Pull request by kedif**
**When you open your pull requests, remove the lines that start and end with underscores (\_)**

## Changes

1. added s&p repo link to note

## Purpose
to take a close at our standard and practice 

## Approach
Added the link to the README.md

## Pre-Testing TODOs
Links are not 404ing

## Testing Steps
Make sure links go to right page

## Learning
None ATM

Closes #6

### CODE REVIEW

**Review by demewozshiferaw — APPROVED** [review]
thank you this is a great link

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # DevOps-Apprenticeship-Notes
 
 
+* https://www.youtube.com/watch?v=0yWAtQ6wYNM
+* Hands-On Tutorials for Amazon Web Services (AWS)
+
+
+
```
Comment (michaelachrisco) [inline README.md:8]: I dont think we need this much whitespace within the markdown. Markdown, by default, removes the extra whilespace. 

This is not a blocker, just thought I would mention it.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # DevOps-Apprenticeship-Notes
 
 
+* https://www.youtube.com/watch?v=0yWAtQ6wYNM
+* Hands-On Tutorials for Amazon Web Services (AWS)
```
Comment (michaelachrisco) [inline README.md:5]: How do you feel about making this one line? For example: 
```suggestion
* Hands-On Tutorials for Amazon Web Services (AWS): https://www.youtube.com/watch?v=0yWAtQ6wYNM
```

**Review by michaelachrisco — APPROVED** [review]
Awesome job @kedif ! I see that you added a great description, made the branch to our standard, and made a contribution that is to a good video.ed it yet, so ill take a look at some later date. NJ! 

Two non-blockers:
1. Extra whitespace is removed from Markdown by default. You may want to remove those extra lines to make the markdown more maintainable in the future. 
2. I believe that having the description and link in the same line will help with making the note page more readable. 

Thanks for the contribution!

**Comment by michaelachrisco** [comment]
Since this PR is a bit on the older side, I would suggest closing the PR and remaking it with your changes here. Its the easiest way of dealing with the file conflicts. Alternatively, we could go over how to deal with git commit conflicts.

**Pull request closed without merging**